sdk/confidentialledger/azure-security-confidentialledger/swagger/README.md

```diff
@@ -1,4 +1,6 @@
 ## Generate autorest code
+### To run, use `autorest --tag:confidential-ledger README.md`
```

Comment:
Add a similar note to the identity (certificate) service below. `--tag:confidential-ledger-identity`. The tag should also be updated to `certificate` instead of `identity`.